NodeManager.h

```diff
@@ -206,6 +219,10 @@
   // BH1750 sensor, return light in lux
   #define SENSOR_BH1750 16
 #endif
+#if MODULE_TSL2561 == 1
+  // TSL2561 sensor, return light in lux
+  #define SENSOR_TSL2561 28
```

Comment:
Sorry, i made it too quickly, as i updated manually based upon my table that didn't include all these new sensors (sonoff, bosch...)